imports/components/header/index.js

```diff
@@ -188,3 +176,20 @@ export default class Header extends Component {
     );
   }
 }
+
+export default connect((state) => ({
```

Comment:
@johnthepink I'd prefer to see the reducer as its own function that can be exported and tested. That way we can easily verify the mapping of the state within connect